config/locales/en.yml

```diff
@@ -20,6 +20,7 @@ en:
       no_changes: "No changes made"
       account_updated: "Your account has been updated"
       none_selected: "Please select a user to edit"
+      inactive_user: "Success! The user's account has been marked as inactive"
```

Comment:
But we'll probably need to talk about how to handle the confirmation message when we implement bulk delete and want to reuse this object to do so.